### PR TITLE
Add F2Pool

### PR DESCRIPTION
**Pull request by Yang-Nankai**
F2Pool is one of the largest cryptocurrency mining pools in the world, supporting the mining of a wide range of digital currencies, including Bitcoin, Ethereum, and Litecoin. It provides tools and resources for miners to optimize their mining operations, monitor performance, and manage payouts. F2Pool is designed to offer reliable and efficient mining services with a global server network. The platform offers 2FA for the account login verification process. The website is ranked 29,614 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=f2pool.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Comment by Yang-Nankai** [comment]
image

**Comment by Yang-Nankai** [comment]
Please forgive me for not finding a suitable SVG Logo.

File: entries/f/f2pool.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "F2Pool": {
+    "domain": "f2pool.com",
+    "img": "f2pool.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://f2pool.zendesk.com/hc/articles/360058722731",
+    "categories": [
+      "cryptocurrencies"
+    ]
```
Comment (hkamran80) [inline entries/f/f2pool.com.json:18]:
```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

Per the [Terms of Service](https://www.f2pool.com/terms) (emphasis mine):
> NOTICE ON PROHIBITED USE - RESTRICTED PERSONS: THE SERVICES ARE NOT OFFERED TO AND MAY NOT BE USED BY PERSONS OR ENTITIES WHO: (A) RESIDE IN, ARE CITIZENS OF, ARE LOCATED IN, ARE INCORPORATED IN, OR HAVE A REGISTERED OFFICE IN **ANY STATE, COUNTRY OR OTHER JURISDICTION THAT IS DESIGNATED BY THE U.S. SECRETARY OF STATE AS A COUNTRY SUPPORTING INTERNATIONAL TERRORISM**, OR THAT IS SUBJECT TO **ECONOMIC SANCTIONS** SUCH THAT U.S. NATIONALS CANNOT LAWFULLY ENGAGE IN TRANSACTIONS AS DESIGNATED BY THE **OFFICE OF FOREIGN ASSETS CONTROL (OFAC)**, OR THAT IS SUBJECT TO UNITED KINGDOM SANCTIONS (AS EXTENDED TO THE BRITISH VIRGIN ISLANDS) (EACH SUCH JURISDICTION, A “RESTRICTED JURISDICTION”); OR (B) LISTED ON THE OFAC SPECIALLY DESIGNATED NATIONALS, BLOCKED PERSONS LIST, OR ANY OTHER CONSOLIDATED PROHIBITED PERSONS LIST AS DETERMINED BY ANY APPLICABLE GOVERNMENTAL AUTHORITY INCLUDING THE UNITED KINGDOM SANCTIONS LIST (AS EXTENDED TO THE BRITISH VIRGIN ISLANDS) (EACH SUCH PERSON OR ENTITY, A “RESTRICTED PERSON”). WE DO NOT MAKE EXCEPTIONS.

**Review by hkamran80 — CHANGES_REQUESTED** [review]
> Please forgive me for not finding a suitable SVG Logo.

No problem!

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**